geometry_msgs/Vector3 car_location
geometry_msgs/Vector3 milestone
voronoi/LineSegment[] obstacles